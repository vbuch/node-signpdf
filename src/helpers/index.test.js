import * as helpers from './index';

describe('Helpers index', () => {
    it('Exports expected helpers', () => {
        expect(Object.keys(helpers).sort()).toMatchSnapshot();
    });
});
