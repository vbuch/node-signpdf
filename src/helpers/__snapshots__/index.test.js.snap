// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Helpers index Exports expected helpers 1`] = `
Array [
  "extractSignature",
  "findByteRange",
  "pdfkitAddPlaceholder",
  "plainAddPlaceholder",
  "removeTrailingNewLine",
]
`;
